db/helpers/utils.js

```diff
@@ -20,3 +24,15 @@ exports.formatComments = (comments, idLookup) => {
     };
   });
 };
+
+
+exports.checkExists = (table, column, value) => {
+  // %I is an identifier in pg-format
+  const queryStr = format('SELECT * FROM %I WHERE %I = $1;', table, column);
+  return db.query(queryStr, [value]).then(( { rows }) => {
+  if (rows.length === 0) {
+    // resource does NOT exist
+    return Promise.reject({ status: 404, msg: 'Resource not found' });
+  }
+})
+};
```

Comment:
Purely an opinion - but I might suggest this new function exist in another file away from the seeding util functions - maybe a models/utils.js?
Also, wonderful generalised function here!